plugins/osdn/multitenant/multitenant.go

```diff
@@ -31,6 +31,9 @@ func Master(osClient *osclient.Client, kClient *kclient.Client, clusterNetwork s
 	if err != nil {
 		glog.Fatalf("SDN initialization failed: %v", err)
 	}
+	// TODO: Get admin namespaces from master node config
```

Comment:
In terms of of multi-tenant networks where tenants are defined by namespaces/projects, admin namespaces are ones which should be allowed to reach all networks. e.g. a load-balancer should be in an admin namespace (and hence a network) that can reach any pod in any network (namespace).